config.go

```diff
@@ -60,6 +71,40 @@ func (cfg Config) MakeLabels(types []string) string {
 	return labels
 }
 
+func (cfg Config) PropertyMap(tp, val string) interface{} {
+	if _, exists := cfg.PropertyTypeMappings[cfg.Map(tp)]; exists {
+		cnvrt := cfg.PropertyTypeMappings[cfg.Map(tp)]
```

Comment:
If type mapping exists, call ls.GetValueAccessor(cfg.PropertyTypeMappings[cfg.Map(tp)])
Then use that to convert value. You should not be doing switch-case.